cocotb/drivers/__init__.py

```diff
@@ -55,6 +55,8 @@ def start(self, generator=None):
 
         Args:
             generator (optional): Generator yielding data.
+            Generator should return a tuple with the number of cycles 
+            to be on, followed by the number of cycles to be off.
```

Comment:
`return a tuple` &rarr; `yield tuples`.


It might be good to give an example, like:
```
bit_driver.start((1, i % 5) for i in itertools.count())
```
and add something like "typically the generator should go on forever"

Reply:
As an aside - it doesn't make any sense to me that `BitDriver(sig, clk, gen_a).start(gen_b)` is legal

Reply:
I'll add the documentation. I don't feel confident enough to get the sanity check right.